process_labels/loading_funcs/tanzania.py

```diff
@@ -85,3 +89,85 @@ def load_tanzania():
     df = df.reset_index(drop=True)
     df[RequiredColumns.INDEX] = df.index
     return df
+
+
+def load_tanzania_ecaas():
+
+    ecaas_files = (DATASET_PATH / "tanzania" / "tanzania_rice_ecaas").glob("*.csv")
+
+    gdfs: List[geopandas.GeoDataFrame] = []
+    for file_path in ecaas_files:
+        gdf = geopandas.GeoDataFrame(crs="EPSG:32736")
```

Comment:
@adebowaledaniel , is the coordinate system of the original data actually in `EPSG:32736`? Or is it in lat / lon?

- If its originally in `EPSG:32736`, then it needs to be converted before you assign it to the latitude and longitude columns (i.e. `gdf[RequiredColumns.LAT] = df["consent_given/_field_center_latitude"]`
- If its in lat/lon then the conversion at the end means the `geometry` column will be incorrect.

Reply:
@gabrieltseng, It's in lat/lon; I believe it is fixed now.